src/components/TweetCreationForm.tsx

```diff
@@ -0,0 +1,159 @@
+import { OverlayTrigger, Popover } from "react-bootstrap";
+import {
+  EmojiSmile,
+  Image as ImageIcon,
+  Globe2,
+  CheckLg,
+  PersonCheck,
+  At,
+} from "react-bootstrap-icons";
+import styled from "styled-components";
+
+const StyledCloseButton = styled.button`
```

Comment:
Bootstrap has a close button, maybey better use it?

Reply:
I have use bootstrap, styled is for hover. Only replace 'border-none'